src/test/clojurescript/clara/test.cljs

```diff
@@ -12,4 +12,4 @@
     (println "FAIL")))
 
 (defn ^:export run []
-  (test/run-tests 'clara.test-rules 'clara.test-common))
+  (test/run-tests 'clara.test-rules 'clara.test-common 'clara.test-salience))
```

Comment:
I checked Travis and `test-salience` did run but I can see I broke the pattern here of requiring the test file at the top of the ns...